scraper/subjects.json

```diff
@@ -0,0 +1,754 @@
+[
```

Comment:
We need those so that the subjects that we manually changed in order to prevent situations where the subjects short names are equal. Otherwise the scraper will re-generate those files every time it runs, which both makes it a lot slower and generates filters with problems like duplicated names.